### PR TITLE
Add script to check memory quota and usage for a single org

### PR DESCRIPTION
**Pull request by mheadd**
## Changes proposed in this pull request:
- Add script to check the org memory quota and current usage for a single org.
- Useful when fielding requests to increase or decrease memory quotas for an org.

## security considerations
None

### CODE REVIEW

**Comment by mheadd** [comment]
I use this script a lot when we get requests to increase or decrease memory quotas. Way faster than logging into Stratos. Thought it might be helpful for others.

File: audit/get-org-mem-usage.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  then
+    echo "You must supply an org name"
+    exit 1
+fi
+
+# Get organization details
+org=$(cf curl v3/organizations/?names=$1)
+
+# Get the guid for the organization
+guid=$(echo $org | jq '.resources[] | .guid' | sed 's/\"//g')
```
Comment (apburnes) [inline audit/get-org-mem-usage.sh:31]: You can use the `-j` flag with jq to remove the quotes

**Review by apburnes — APPROVED** [review]
Looks good! Added a comment about using a flag so you don't have to clean up the quotes with `sed 's/\"//g'`

File: audit/get-org-mem-usage.sh (hunk 1 above)
Comment (mheadd) [reply audit/get-org-mem-usage.sh]: Good call, @apburnes. Change made.

**Pull request merged**